contracts/OtcEscrowApprovals.sol

```diff
@@ -25,8 +25,12 @@ contract OtcEscrowApprovals {
     uint256 public immutable balAmount;
     uint256 public immutable aaveAmount;
 
+    bool private hasSwapOccured;
```

Comment:
don't see a reason to make this private. it should be easy to check if the swap occurred or not for anyone

Reply:
Done